include/libdnf/rpm/repo.hpp

```diff
@@ -36,6 +36,8 @@ class Base;
 
 namespace libdnf::rpm {
 
+class SolvSack;
```

Comment:
In some cases we added commend that this is a forward declaration. Shall we used everywhere?

Reply:
I don't know. The comment has none information value for me.
```
// forward declarations
class SolvSack;
```

For me is the same as:
```
// assign values
a = 5;
```
Maybe there was an attempt to mark one place in the code for all forward declarations. So, if you want, I can add comment "forward declarations"